calyx-py/test/correctness/queues/binheap.py

```diff
@@ -29,107 +31,369 @@ def insert_swap(prog, name, width, len, idx_w):
     return comp
 
 
-def insert_binheap(prog, name):
+def insert_binheap(prog, name, factor):
```

Comment:
Nice, thanks for generalizing to the `queue_len_factor` style of determining the length. Perhaps consider a slightly more descriptive variable name, and a brief explanation somewhere of how `factor` determines the heap's capacity. Just saves the next person a little confusion :)